### ISSUE TITLE
cyanite-prune blowing up

### ISSUE THREAD

**Issue by jforest** [WrathOfChris/cyanite-utils#12]
I can do cyanite-list just fine, but prune blows up

```
[jforest@cyanite3.support.aws ~]$ cyanite-prune -v
Traceback (most recent call last):
  File "/usr/bin/cyanite-prune", line 9, in <module>
    load_entry_point('cyanite-utils==0.0.11', 'console_scripts', 'cyanite-prune')()
  File "/usr/lib/python2.7/site-packages/cyanite_utils/cli.py", line 87, in cyanite_prune
    metrics = CyaniteMetrics(config)
  File "/usr/lib/python2.7/site-packages/cyanite_utils/CyaniteMetrics.py", line 21, in __init__
    for (rollup, interval) in self.config.rollups():
  File "/usr/lib/python2.7/site-packages/cyanite_utils/Config.py", line 63, in rollups
    rollups.append((r['period'], r['rollup']))
TypeError: string indices must be integers, not str
```

I did a pip install:
```
cyanite-utils==0.0.11
```

**Comment by WrathOfChris** [comment]
Ahh, I haven't had time yet to implement the alternate [carbon-style](http://graphite.readthedocs.org/en/latest/config-carbon.html#storage-schemas-conf) notation (ie: "10s:1d" )

If your rollups are declared like this, it should work:
```
  rollups:
    - period: 60480
      rollup: 10
    - period: 105120
      rollup: 600
```

**Comment by jforest** [comment]
Ok, changing that temporarily does let it work.  Good to know!  Thanks for the quick response!

**Comment by jforest** [comment]
I created https://github.com/WrathOfChris/cyanite-utils/pull/13 to help with this!

**Comment by WrathOfChris** [comment]
Just merged #13 -- Thanks @jforest
